# Robot name goes here
urdf_file:
  # default controller
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Controller 1
  left_wheel_steering_controller:
    type: effort_controllers/JointPositionController 
    joint: rotator_left_joint  
    pid : {p: 112 , i: 0.0 , d: 0.2} 

  # Controller 2
  right_wheel_steering_controller: 
    type: effort_controllers/JointPositionController 
    joint: rotator_right_joint  
    pid : {p: 112 , i: 0.0 , d: 0.2}
    
  # Controller 3
  left_wheel_controller:
    type: velocity_controllers/JointVelocityController 
    joint: front_left_wheel_joint 
    pid : {p: 112 , i: 0.0 , d: 0.2}
 
# Controller 4
  right_wheel_controller: 
    type: velocity_controllers/JointVelocityController 
    joint: front_right_wheel_joint 
    pid : {p: 112 , i: 0.0 , d: 0.2}

